taskw/warrior.py

```diff
@@ -412,10 +412,10 @@ class TaskWarriorShellout(TaskWarriorBase):
     and https://github.com/ralphbean/taskw/issues/30 for more.
     """
     DEFAULT_CONFIG_OVERRIDES = {
+        'verbose': 'nothing',
```

Comment:
Unfortunately, dictionaries aren't ordered by insertion order in Python < 3.7; so I'm afraid this doesn't quite work unless you also change this to be a `collections.OrderedDict` instance.

Reply:
Python 3.6 support ends in 7 months (23 Dec 2021). So while incomplete, this simple change fixes `taskw` for majority of supported Python versions.